.github/workflows/review-deploy.yml

```
@@ -0,0 +1,126 @@
+name: Review Deployment
+
+on:
+  push:
+    branches:
+      - "MP-1889-review-cloud-function"
+
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: "0 */24 * * *"
```

Nit: This is flagged as aninvalid cron string by my local linter. Isn't this equivalent to this?
```suggestion
    - cron: '0 0 * * *'
```